src/api/app/assets/stylesheets/webui/application/cloud.scss

```diff
@@ -0,0 +1,33 @@
+.cloud_button {
+  padding: 20px;
+  height: 150px;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+  &:hover {
+    background-color: #f2f2f2;
+  }
+
+  &.disabled {
+    background-color: #f2f2f2;
+    opacity: 0.4;
+  }
+}
+
+.cloud_grid {
```

Comment:
Please do this with [960gs](https://960.gs/), no need to introduce something new...

Reply:
Done :+1: